src/screens/ProfileScreen.js

```diff
@@ -0,0 +1,161 @@
+import React, { Component } from "react";
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  SafeAreaView,
+  TouchableOpacity,
+  Linking
+} from "react-native";
+import { Header } from "react-native-elements";
+import { SimpleLineIcons, Ionicons } from "@expo/vector-icons";
+
+export default class Profile extends Component {
+  state = {
+    profile: {},
+    error: ""
+  };
+  componentDidMount() {
+    this.getProfile();
+  }
+  toHome = () => {
+    const { navigation } = this.props;
+    navigation.goBack();
+  };
+
+  getProfile = () => {
+    const { navigation } = this.props;
+    const username = navigation.getParam("username");
+    const url = `https://api.github.com/users/${username}`;
+
+    fetch(url)
+      .then(res => res.json())
+      .then(res => {
+        this.setState({
+          profile: res,
+          error: res.message || null,
+          loading: false
+        });
+      })
+      .catch(error => {
+        this.setState({ error: error });
+      });
+  };
+
+  toGitHub = () => {
+    const { navigation } = this.props;
+    const username = navigation.getParam("username");
+    const url = `https://github.com/${username}`;
+    Linking.openURL(url);
+  };
+
+  render() {
```

Comment:
![Screenshot 2019-06-07 at 14 47 45](https://user-images.githubusercontent.com/6739804/59101935-3f4f2680-8933-11e9-8079-a98c07b06ab0.png)

Code Climate seems to be flagging your render method. 

Kindly research on best practice when it comes to render function LOC requirements.

Reply:
Thank you! I am going to research and refactor later.